docker-compose.yml

```diff
@@ -24,7 +24,7 @@ services:
       
   celery:
     <<: *api-common
-    command: celery -A main worker --loglevel=info
+    command: celery -A main worker --loglevel=info --statedb data/celery-worker.state
```

Comment:
Is this just storing worker state?

Reply:
Yeah! I wasn't thinking we needed to keep track of any state or stored results, etc., because we don't care about retrieving any stored result from our tasks. BUT the `revoke()` method to terminate a task doesn't literally kill the worker process that's running the task - it just sends the worker(s) a message to "ignore/stop running the task with this ID, when you next encounter it". I think our celery worker was previously getting these revocation messages, but instantly forgetting them due to not having any sort of place to remember things...and that was why our task termination wasn't working. (https://docs.celeryq.dev/en/stable/userguide/configuration.html#worker-state-db)